helm/hpcc/templates/_helpers.tpl

```diff
@@ -808,6 +808,7 @@ Generate instance queue names
   prefix: {{ .prefix }}
   {{- end }}
   queriesOnly: true
+  ldapUser: {{ .ldapUser }}
```

Comment:
@richardkchapman what would be your preferred name for this option.  Kevin had systemUser.  It isn't really a system user or ldap user (security may not use ldap).  I think it is the user that is used for all file lookups from roxie.  I don't have a strong opinion any way.

Reply:
It's called ldapUser on bare metal legacy configs, I think. So probably makes sense to use the same name?